client/src/components/App.jsx

```diff
@@ -18,11 +18,16 @@ import About from './About';
 import Disclaimer from './Disclaimer';
 import NotFoundPage from './NotFoundPage';
 import actions from '../redux/actions/actions';
+// import { fetchDemographicsComorbidities } from '../redux/actions/onboarding'
```

Comment:
Remove this comment and others in file.

Also, why did you opt to remove this here?

Reply:
So I actually didn't remove this, I added it and then commented it -- my bad for leaving it there without addressing it, it was outside the scope of this issue but it was something I noticed in the course of working on this -- that we were adding this detail about the individual into gaia and then never using it, I wasn't sure if we also wanted that in redux so I wrote that thunk that we could add if we want that data available.

Reply:
I can just remove it from this PR, I left the comment in more to remind myself.